deck-functional-tests/docker/Dockerfile.local

```diff
@@ -0,0 +1,7 @@
+FROM node:8-stretch
+LABEL last_update="2018-10-01 13:31"
```

Comment:
what's this date?

Reply:
I've removed this.  I added it originally as a quick way to invalidate the Docker build cache. When the date is updated all subsequent steps are re-run even if the contents of those commands are exactly the same.  No longer necessary I don't think since there's a `docker build --no-cache` flag that I wasn't originally aware of.